package.json

```
@@ -56,8 +58,15 @@
     ]
   },
   "prettier": {
+    "importOrder": [
+      "^react$",
+      "<THIRD_PARTY_MODULES>",
+      "^[.]"
```

I changed it so the typescript engine prefers auto-importing with the path alias set in the `tsconfig.json`. the import order should be as follows:
```suggestion
      "<BUILTIN_MODULES>",
      "^react$",
      "<THIRD_PARTY_MODULES>",
      "^@/.*$",
      "^[.]"
```

---

also, make sure to run `npm run format` so everything gets updated. I'll look into adding CI for this stuff